llvm/lib/Transforms/Vectorize/LoopVectorizationPlanner.h

```diff
@@ -161,6 +161,15 @@ class VPBuilder {
     return tryInsertInstruction(
         new VPInstruction(Opcode, Operands, WrapFlags, DL, Name));
   }
+
+  VPInstruction *createFPOp(unsigned Opcode,
+                            std::initializer_list<VPValue *> Operands,
+                            DebugLoc DL = {}, const Twine &Name = "",
+                            FastMathFlags FMFs = {}) {
+    auto *Op = new VPInstruction(Opcode, Operands, FMFs, DL, Name);
+    return tryInsertInstruction(Op);
```

Comment:
> consistency nits: these are chained together into one return tryInsertInstruction(new VPInstruction(...)); above and below, if not replaced by a suitable createInstruction()?

Adjusted, thanks

> createSelect() below also supports FMFs, but via optional - should it too default to {} instead?

IIRC this is because FMFs can only be set for selects with FP operands, which should be clearer via std::optional